packages/db/src/config.ts

```diff
@@ -120,6 +130,16 @@ export const getKnexConfig = async ({
     connection: await getConnectionConfig({ env, secretsManager }),
     debug: env.KNEX_DEBUG === 'true',
     asyncStackTraces: env.KNEX_ASYNC_STACK_TRACES === 'true',
+    pool: {
+      min: 0,
+      max: Number.parseInt(env.dbMaxPool ?? '2', 10),
+      idleTimeoutMillis: Number.parseInt(env.idleTimeoutMillis ?? '1000', 10),
```

Comment:
I mapped these to match the config from tarn, however am open to suggestions on a better user-facing name.